java-grupa8/homework-1/tema/src/main/java8/ro/siit/tema/InfoInit.java

```diff
@@ -0,0 +1,75 @@
+package ro.siit.tema;
+
+/**
+ * @author x-RauL-x
+ * A class used to initialize all the SalesRepresentatives in the SalesDepartment class
+ */
+
+public class InfoInit {
```

Comment:
This code is part of another task(homework). It should not show up in this pull request.